### ISSUE TITLE
request for new common features - withdraw

### ISSUE THREAD

**Issue by heavymanto** [JKorf/CryptoExchange.Net#173]
I think withdrawal is a very useful feature on a common interface as it is typical to move funds between different exchanges. 
Thank you for great work ;)

**Comment by JKorf** [comment]
I agree, but I'll have to look at if this is doable. I think some exchanges require quite specific things for withdrawal

**Comment by JKorf** [comment]
It's been a while, but this is available now, see https://jkorf.github.io/CryptoExchange.Net/#idocs_shared

**Comment by heavymanto** [comment]
wow Jan, I can't wait to try out these new features. Thanks for your great
job!

Il giorno mer 9 ott 2024 alle ore 09:53 Jan Korf ***@***.***>
ha scritto:
